modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/side_panel_content/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	CLOSE_SIDE_PANEL,
+	OPEN_MODAL,
+	OPEN_MODAL_FROM_IFRAME,
+} from 'frontend-taglib-clay/data_set_display/utils/eventsDefinitions';
+
+function initializeSidePanelListeners() {
+	Liferay.on(OPEN_MODAL, (payload) => {
+		window.top.Liferay.fire(OPEN_MODAL_FROM_IFRAME, payload);
+	});
+
+	document.querySelector('body').classList.remove('open');
+
+	document
+		.querySelectorAll('.side-panel-iframe-close, .btn-cancel')
+		.forEach((trigger) => {
+			trigger.addEventListener('click', (e) => {
+				e.preventDefault();
```

Comment:
[Best not to abbreviate](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/naming.md#avoid-abbreviations) (ie. `e` → `event`).

Reply:
Omg, I'm really terrible with this rule. Sorry guys

Reply:
I wouldn't worry too much about this. We can actually lint for the common cases (eg. `e`, `evt` and friends) and then you won't have to worry about it. I thought we might have an issue for that already but I couldn't find it, so I created [this one](https://github.com/liferay/liferay-frontend-projects/issues/437).